camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/SpecParameterParsingTest.java

```diff
@@ -66,19 +66,20 @@ public void testYamlInputsParsed() {
                 "    - simple",
                 "    - name: explicit_name",
                 "    - name: third_input",
-                "      type: integer");
+                "      type: integer",
+                "      pinned: true");
         EntitySpec<?> item = mgmt().getTypeRegistry().createSpec(mgmt().getTypeRegistry().get(itemId), null, EntitySpec.class);
         List<SpecParameter<?>> inputs = item.getParameters();
         assertEquals(inputs.size(), 6);
         SpecParameter<?> firstInput = inputs.get(0);
         assertEquals(firstInput.getLabel(), "simple");
-        assertEquals(firstInput.isPinned(), true);
+        assertEquals(firstInput.isPinned(), false);
```

Comment:
How come the tests still pass with this change? Am I missing something?
The default is flipped to `true` now, so the change shouldn't be needed?